addon/components/attach-popover.js

```diff
@@ -39,6 +40,7 @@ export default Component.extend({
     }
   }),
   class: DEFAULTS.class,
+  style: DEFAULTS.style,
```

Comment:
Bikeshed: Let's keep these alphabetized by moving this to the bottom of this property list.